KafkaIngester/src/main/java/io/deephaven/kafka/KafkaTools.java

```diff
@@ -0,0 +1,77 @@
+package io.deephaven.kafka;
+
+import io.deephaven.UncheckedDeephavenException;
+import io.deephaven.db.tables.Table;
+import io.deephaven.db.tables.TableDefinition;
+import io.deephaven.db.tables.utils.ColumnsSpecHelper;
+import io.deephaven.db.tables.utils.DBDateTime;
+import io.deephaven.db.v2.utils.DynamicTableWriter;
+import io.deephaven.internal.log.LoggerFactory;
+import io.deephaven.io.logger.Logger;
+import io.deephaven.kafka.ingest.ConsumerRecordToTableWriterAdapter;
+import io.deephaven.kafka.ingest.KafkaIngester;
+import io.deephaven.kafka.ingest.SimpleConsumerRecordToTableWriterAdapter;
+import org.apache.kafka.clients.consumer.ConsumerRecord;
+import org.jetbrains.annotations.NotNull;
+
+import java.io.IOException;
+import java.util.Properties;
+
+public class KafkaTools {
+
+    public static final String KAFKA_PARTITION_COLUMN_NAME = "kafkaPartition";
+    public static final String OFFSET_COLUMN_NAME = "offset";
+    public static final String TIMESTAMP_COLUMN_NAME = "timestamp";
+    public static final String KEY_COLUMN_NAME = "key";
+    public static final String VALUE_COLUMN_NAME = "value";
+    public static final String KEY_DESERIALIZER_PROPERTY = "key.deserializer";
+    public static final String VALUE_DESERIALIZER_PROPERTY = "value.deserializer";
+    public static final String DEFAULT_DESERIALIZER = "org.apache.kafka.common.serialization.StringDeserializer";
+
+    private static final Logger log = LoggerFactory.getLogger(KafkaTools .class);
+
+    public static Table simpleConsumeToTable(
```

Comment:
This is intended for our python/groovy integration.  I called this KafkaTools to follow the pattern of TableTools and ParquetTools.  This file is the basis for all the other changes in this PR that wire things up to be callable from python.

The implementation happens to be here right now because it needed to be somewhere, but likely as this stuff grows it should move somewhere (eg, ParqutTools calls to pretty big files for ParquetTableWriter and such in different, parquet specific modules.  Since we want this stuff to be based on other stuff and not TableWriter, this implementation is scaffolding for integration right now, it is fully expected to change, etc.

Reply:
I meant "Consume -> Consumer"; but it doesn't take a consumer.  It just reads a bit off.  KafkaTools is fine.